R/archive.R

```diff
@@ -599,8 +599,8 @@ epi_archive =
             }
               
             # If a custom time step is specified, then redefine units 
-            before_num = n-1
-            if (!missing(time_step)) before_num = time_step(n-1)
+            before_num = before-1
```

Comment:
Should this be `before_num = before`?

Reply:
I remember discussing this with someone, and that this is not a bug despite my thinking it is. Thus, please confirm it is a bug. For example, if `before = 1` with one-day latency, we get nothing with the slide.

Reply:
As far as I know, switching to `before_num = before` is the desired behavior.